openfisca_us/parameters/gov/states/mo/tax/income/rates.yaml

```diff
@@ -0,0 +1,68 @@
+description: The tax rates for individual residents in Missouri. Thresholds are <=.
+brackets:
+- rate:
+    2021-01-01:
+      value: 0.015  
```

Comment:
We can just have date: x, we don't need to have date: value: x

Reply:
got it!